AdyenCard/Validators/CardSecurityCodeValidator.swift

```diff
@@ -8,8 +8,7 @@
 import Foundation
 
 /// Validates a card's security code.
-@_spi(AdyenInternal)
-public final class CardSecurityCodeValidator: NumericStringValidator, AdyenObserver, StatusValidator {
+public final class CardSecurityCodeValidator: NumericStringValidator, AdyenObserver {
```

Comment:
maybe move `AdyenObserver ` in SPI ?

Reply:
Hmm, `AdyenObserver` is public anyway, what benefit would that have?

Reply:
No direct benefits, just makes public part clear.

`AdyenObserver ` was created back in a days when iOS 13 was not available and we were not able to use `Combine`. Nowadays we can, so this implementation can change..